src/jit/importer.cpp

```diff
@@ -9409,45 +9407,6 @@ void              Compiler::impImportBlockCode(BasicBlock * block)
                         
             goto APPEND;
 
-#else // !_TARGET_X86_ && !_TARGET_ARM_
-
-            // Import this just like a series of LDARGs + tail. + call + ret
```

Comment:
I would suggest to not to delete the code under #else.  It serves as an alternate and quick impl for "jmp method". Instead just enable GT_JMP node generation for ARM64 as well using right #ifdef on line 9395  

e.g.
#if defined(_TARGET_XARCH_) || defined(_TARGET_ARMARCH_)